var/spack/repos/builtin/packages/plasma/remove_absolute_mkl_include.patch

```diff
@@ -0,0 +1,55 @@
+--- a/compute/zgbtrf.c    2017-01-13 13:59:46.000000000 -0500
```

Comment:
Master doesn't need that patch anymore:
https://bitbucket.org/icl/plasma/src/3bd97977c3cf7e6bc2bf6194207d8c0f23e0d0a3/compute/zgbtrf.c?at=default&fileviewer=file-view-default